recipes/21cmFAST/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "21cmFAST" %}
+{% set version = "3.0.0.dev5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0acf356e74ad8a2254f6c729b06b98f4b6cdde0b838e46c614d9ae68f8e56791
+
+
+build:
+  number: 0
+  script: "INC=$(CONDA_PREFIX)/include LIB=$(CONDA_PREFIX)/lib {{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+  
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - pip
+    - cffi
+    - fftw
+    - gsl
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - libopenblas
```

Comment:
Please see docs on how to specify blas.

Reply:
I'm finding the docs a bit hard to understand here -- my impression is that I should just have `libblas` in the `host` section, and nowhere else. Does that seem right?

Reply:
Yes, that's right. You also need `libcblas` if the package is using cblas symbols and `liblapack` if the package is using lapack symbols.

Reply:
OK, thanks. I *think* it just needs `blas`, and that is now fixed.

Reply:
All runs OK so it looks like that was correct.